### PR TITLE
Readability unit tests

### PR DESCRIPTION
**Pull request by peteruithoven**
An attempt to make the unit tests more readable / understandable.

### CODE REVIEW

File: test/index.spec.js [hunk 1]
```diff
@@ -4,55 +4,71 @@ import undoable, { ActionCreators, excludeAction, includeAction, isHistory } fro
...
+    past: [5, {}, 3, null, 1],
+    present: Math.pow(2, 32),
+    future: []
+  }
+})
 
 // Test undoable reducers as a function of a configuration object
 // `label` describes the nature of the configuration object used to run a test
-function runTestWithConfig (testConfig, initialStoreState, label) {
+function runTests (label, { unduableConfig, initialStoreState, testConfig } = {}) {
```
Comment (omnidan) [inline test/index.spec.js]: small typo here, should be `undoableConfig`

Comment (omnidan) [inline test/index.spec.js]: what's `testConfig` now? is it even used?

Comment (peteruithoven) [inline test/index.spec.js]: It's to add test specific test config, like which actions are excluded. I thought it made more sense than adding it to the config which is also used as undoable config.

**Comment by pl12133** [comment]
Looks great!

**Pull request merged**